src/main/webapp/assets-lti/js/controllers/controllers.js

```diff
@@ -14,20 +14,15 @@ canvasSupportApp.controller('courseController', ['Course', 'Courses', 'Sections'
     Sections.getSectionsForCourseId($scope.course.id).then(function (result) {
       $scope.loadingSections = false;
       $scope.course.sections =_.sortBy(result.data, 'name');
-    });    
+      $scope.currentTermSISID = $scope.course.sections[0].sis_course_id.substring(0, 4);
+      /* adds to the scope a list of sections (by sis_section_id) that the current user can perform actions on */
+      var mPathwaysCoursesUrl = 'manager/mpathways/Instructors?instructor=' + $rootScope.ltiLaunch.custom_canvas_user_login_id +'&termid=' + $scope.currentTermSISID;
+      Course.getMPathwaysCourses(mPathwaysCoursesUrl, $scope.currentTermSISID).then(function (result) {
+        $scope.mpath_courses = result;
```

Comment:
If `getMPathwaysCourses()` fails, will `.then()` be called?  If not, what value will `$scope.mpath_courses` have?